cogs/helpers.py

```diff
@@ -38,18 +36,61 @@
 import re
 import math
 import time
-import os
 import datetime
+import pickle
+import feedparser
 from .utils.paginator import Pages
 from .utils.requests import fetch
 
 MCGILL_EXAM_URL = "https://www.mcgill.ca/exams/dates"
+CFIA_FEED_URL = "http://inspection.gc.ca/eng/1388422350443/1388422374046.xml"
 
 
 class Helpers():
     def __init__(self, bot):
         self.bot = bot
 
+    async def cfia_rss(self):
+        # Written by @jidicula
+        """
+        Co-routine that periodically checks the CFIA Health Hazard Alerts RSS
+         feed for updates.
+        """
+        await self.bot.wait_until_ready()
+        while not self.bot.is_closed():
+            food_channel = utils.get(
+                self.bot.get_guild(self.bot.config.server_id).text_channels,
+                name="food")
```

Comment:
Maybe this should be a configurable thing so that it isn't hardcoded to a specifically-named channel. Fine for now, but maybe an issue should be created for a future pr

Reply:
Yeah, it should definitely go in config.ini. I'll make another PR for both this and the above suggestion for pickles.